external-notification-service.openApi.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: '3.0.1'
+info:
+  title: UKHO External Notification Service APIs
+  description: "This is a light weight open api spec for UKHO External Notification Service. One of its main purpose is to allow importing ENS APIs in APIM."
+  version: '0.0.1'
+  contact:
+    email: martin.rock-evans@ukho.gov.uk
+    name: Martin Rock-Evans
+servers:
+  - url: http://ens.admiralty.co.uk
```

Comment:
As we are in API would we not have to have a URL like https://admiralty.azure-api.net/ens

Reply:
also I a feeling that if you leave this empty APIM fills this for you.

Reply:
It was more of a placeholder and now I removed it.